hotel/security/hotel_security.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE xml>
+<odoo>
+
+    <data noupdate="1">
```

Comment:
<data> tag is deprecated. You can use <odoo noupdate="1"> now.

Reply:
Yes right indeed. Will amend that, thanks!

@pntabuye Also, can you review the Code functionally and technically of other PRs.

Reply:
I'll start checking tonight @darshan-serpent - I need this one for a client I am upgrading to Odoo 11.